lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -214,7 +219,11 @@ def instructor_dashboard_2(request, course_id):
     if len(openassessment_blocks) > 0 and access['staff']:
         sections.append(_section_open_response_assessment(request, course, openassessment_blocks, access))
 
+    if course_id == 'course-v1:HarvardX+CS50+X':
+        log.info('Investigating log at %s : before Disable Button (calling _is_small_course)', time.time() - start_time)
     disable_buttons = not _is_small_course(course_key)
```

Comment:
no need to have `_is_small_course` method now. you can directly add `CourseEnrollment.objects.is_small_course(course_key)`

Reply:
`_is_small_course` has been completely replaced by `CourseEnrollment.objects.is_small_course(course_key)`